Cargo.toml

```diff
@@ -13,8 +13,13 @@ keywords = ["contour", "polygon", "isoring", "marching-squares", "geojson"]
 license = "MIT/Apache-2.0"
 
 [dependencies]
-geojson = ">=0.16, <=0.24"
+geojson = { version = ">=0.16, <=0.24", optional = true }
```

Comment:
What do you think about making geojson an optional dependency?

And should it be disabled by default?

(happy to change either of these, I just went with my personal preference)